ts/features/itwallet/common/utils/itwClaimsUtils.ts

```diff
@@ -27,29 +25,24 @@ import { ParsedCredential, StoredCredential } from "./itwTypesUtils";
  */
 
 /**
- * Retrieves the organization name from the evidence claim of the given credential.
- * If the evidence claim is not present or cannot be decoded, a fallback value is returned.
- * @param credential - The parsed credential object.
- * @returns The organization name from the evidence claim or a fallback value.
+ * We strongly discurage direct claim manipulation, but some spacial cased must be addressed with direct access
  */
-export const getEvidenceOrganizationName = (credential: ParsedCredential) =>
-  pipe(
-    credential.evidence,
-    O.fromNullable,
-    O.fold(
-      () => I18n.t("features.itWallet.generic.placeholders.organizationName"),
-      evidence =>
-        pipe(
-          evidence.value,
-          EvidenceClaim.decode,
-          E.fold(
-            () =>
-              I18n.t("features.itWallet.generic.placeholders.organizationName"),
-            some => some[0].record.source.organization_name
-          )
-        )
-    )
-  );
+export enum WellKnownClaim {
+  /**
+   * Unique ID must be excluded from every credential and should not rendered in the claims list
+   */
+  unique_id = "unique_ID",
+  /**
+   * Claim used to extract expiry date from a credential. This is used to display how many days are left for
+   * the credential expiration or to know if the credential is expired
+   */
+  expiry_date = "expiry_date",
+  /**
+   * Claim used to display a QR Code for the Disability Card. It must be exlcuded from the common claims list
```

Comment:
Small typo: `exlcuded`!

Reply:
Addressed in https://github.com/pagopa/io-app/pull/6074/commits/b07750f59059602857b02be7907fadd88ca2c35f